src/wanderer/lib.ts

```diff
@@ -104,6 +104,8 @@ export function canWander(location: Location, type: DraggableFight): boolean {
   switch (type) {
     case "backup":
       return canWanderTypeBackup(location);
+    case "pigskinner":
+      return have($skill`Free-For-All`) && canWanderTypeYellowRay(location);
```

Comment:
I don't think we want this logic included in wanderer; we should just have this check in barfTurn, and have pigskinner either use the yellow ray code or the backup code.

Reply:
It's also used to tell whether a Guzzlr quest can be completed in the available turns.

Reply:
Sure, but if we're treating it as identical to backup, for example, we can just change `pigSkinnerRay` to `canWanderCache["backup"] && have($skill`Free-For-All`) ? Math.floor(estimatedGarboTurns() / 25) : 0`

which, upon reflection, we should also have yellowRay check to see if we own parka before counting those. That's out of scope of this PR, just not noticed it.

Reply:
That seems like too low of an estimate since the two are independent.